modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/size_items.dm

```diff
@@ -0,0 +1,108 @@
+/obj/item/clothing/neck/size_collar
+	name = "size collar"
+	desc = "A shiny black collar embeded with technology that allows the user to change their own size. Mysteriously, this only seems to work while inside of dorms."
```

Comment:
Description isn't clear enough since it only works in  the interlink dorms, not just "dorms"